lib/kitchen/driver/opennebula.rb

```diff
@@ -180,7 +180,7 @@ def destroy(state)
       def opennebula_connect()
         opennebula_creds = nil
         if ENV.has_key?('ONE_AUTH')
-          if File.exists(ENV['ONE_AUTH'])
+          if File.exists?(ENV['ONE_AUTH'])
```

Comment:
Missed it in the first place :(
Since we are here let's use File.exist instead (https://ruby-doc.org/core-2.2.0/File.html#method-c-exists-3F)